README.md

```diff
@@ -162,6 +162,17 @@ The `AppDelegate.m` file can only have one method for `openUrl`. If you're also
 
 ## Usage
 
+### SDK Initialization
+
+The `autoInitEnabled` option is removed from [facebook-ios-sdk#v9.0.0](https://github.com/facebook/facebook-ios-sdk/blob/master/CHANGELOG.md#900). On iOS, developers are required to initialize the sdk after app launched.
+
+```js
+import { Settings } from 'react-native-fbsdk';
```

Comment:
This is something that needs to be done earlier in the application or only when using FBSDK?
It's not that clear to me. 
For instance, I use FB only in my sign in page, can I do it only there or would it be better to do the initialization in my app.js?

Reply:
In my experience I think the app will login okay, but will crash if you run a graph request after? That was my experience when I did not put in my AppDelegate `didFinishLaunchingWithOptions` call to the FBSDK, and I **think** (unverifed / open question) `initializeSDK` would have the same effect.

I believe you would want to do it immediately unless you had a GDPR-style opt-in flow which you would have to satisfy prior to using a facebook login anyway I think?, then you would do it immediately after that

Reply:
Yes, that was my point to maybe just to enforce the need to make it happening as earlier as possible in the implementation of their app. 

Unless of course you should obey some GDPR-style flow.

Reply:
@thebergamo I've got your point, let me complete the document.